public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -19,29 +25,88 @@ export class StoryQuestionsQuestion extends React.Component {
     onFormErrorsUpdate: PropTypes.func
   };
 
+  state = {
+    answers: _get(this.props, 'fieldValue.answers', [])
```

Comment:
That expression returns `undefined`, despite there being a default value. Any idea why, @shaundillon ?

Reply:
I think there's a typo, it should be `_.get`

Reply:
I'm importing `'lodash/fp/get'` as done in other modules, isn't that the preferred method?